src/components/core/scrollbars/scrollbars.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { render, screen } from 'test-setup';
+import Scrollbars from './scrollbars';
+import Text from '@core/text/text';
+
+jest.mock('react', () => ({
+    ...jest.requireActual('react'),
+    useState: jest.fn(),
+}));
+
+describe('Scrollbars Component', () => {
+    const setState = jest.fn();
+    (useState as jest.Mock).mockImplementation((init) => [init, setState]);
+    jest.spyOn(React, 'useRef').mockReturnValueOnce({ current: null });
+
+    const props = {
+        autohide: false,
+        dark: false,
+        has_horizontal: false,
+        is_only_horizontal: false,
+        is_only_horizontal_overlay: false,
+        is_scrollbar_hidden: false,
+        has_y_scroll_on_drag_effect: false,
+    };
+
+    it('Scrollbars renders properly', () => {
+        render(
+            <Scrollbars
+                {...props}
+                style={{
+                    width: '300px',
+                    height: '400px',
+                    borderRadius: '10px',
+                    background: props.dark ? '#C15D5D' : '#F7F8F1',
+                }}
+            >
+                {'275577587687785657687567'.split('').map((el, i) => (
```

Comment:
```suggestion
                {'275577587687785657687567'.split('').map((el, i) => (
```
question: 👀  this is a random string to split it to individual text right?

Reply:
@yashim-deriv yes, it's needed to make a column of numbers for simulating content overflow in scrollbars stories 😊:
![Screenshot 2022-04-07 at 10 18 11](https://user-images.githubusercontent.com/87321516/162142233-1e808ef9-e907-4dc8-96a1-a85baad57af6.png)